tests/StackExchange.Redis.Tests/StackExchange.Redis.Tests.csproj

```diff
@@ -30,6 +30,6 @@
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net462' ">
     <Reference Include="Microsoft.CSharp" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.7.1" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="5.0.0" />
```

Comment:
Is this fixing a build error? Definitely one of the danger noodles, so just checking...not worried since it's us and not the lib.

Reply:
just cleaning up package updates; only impacts tests, and I'd rather know that the tests work with "current"

Reply:
kk, gotcha